gpg-interface.c

```diff
@@ -88,6 +88,9 @@ int check_signature(const char *payload, size_t plen, const char *signature,
 
 void print_signature_buffer(const struct signature_check *sigc, unsigned flags)
 {
+	if (flags & GPG_VERIFY_QUIET)
+		return;
```

Comment:
What I mean is that we need a good "reason" as to why gpg-interface, which is git-wide, needs to change in order to make a change to git tag code.